tests/migration_test.go

```diff
@@ -431,7 +431,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 
 				durationLowBandwidth := repeatedlyMigrateWithBandwidthLimitation(vmi, "10Mi", 3)
 				durationHighBandwidth := repeatedlyMigrateWithBandwidthLimitation(vmi, "128Mi", 3)
-				Expect(durationHighBandwidth.Seconds() * 3).To(BeNumerically("<", durationLowBandwidth.Seconds()))
```

Comment:
I'm not sure I understand. The bandwidth is 12 times bigger, but the duration is only 3 times faster? Is this acceptable? Or an indication that something is wrong?

This PR give only one more second and I suspect that it would keep the test close to the edge of failing. Expecting that the duration is 2.5 times faster would give a more consistent result (if we believe it is ok for bandwidth to affect so little, I don't know)

Reply:
> I'm not sure I understand. The bandwidth is 12 times bigger, but the duration is only 3 times faster? Is this acceptable? Or an indication that something is wrong?

As I understand it, the test expects that the high bandwidth duration will be at least 3 times faster than the low bandwidth duration (could be faster). The test was added here https://github.com/kubevirt/kubevirt/pull/6007 maybe @rmohr can help to clarify?

> This PR give only one more second and I suspect that it would keep the test close to the edge of failing. Expecting that the duration is 2.5 times faster would give a more consistent result (if we believe it is ok for bandwidth to affect so little, I don't know)

Yes, this PR only addresses the errors we have found so far, not sure if other values are possible/plausible. It is weird that the failures we had so far were with the high bandwidth migration spending exactly 10s and the low bandwidth migration exactly 30s. Not sure why we had exactly these numbers twice.

Reply:
Do we know (can you log?) what are the typically measured durations? I find it hard to tell when the gap is "too little".

On another note: using equality on floats is a bad practice. If you prefer relaxing the rule by exactly 1, I'd prefer `durationHighBandwidth*3 < durationLowBandwidth + 1`

Reply:
`*2.5` sounds good. I tried to use a value which did not fail after exhaustive runs on my machine (afaik I was far away from the case we see in CI) but still clearly indicated that a bandwidth limitation happened. A factor of `2.5` sounds good too from a validation perspective.

Reply:
> Do we know (can you log?) what are the typically measured durations? I find it hard to tell when the gap is "too little".

Sure, you can check the values in this search https://search.ci.kubevirt.io/?search=migration+with+bandwidth&maxAge=336h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job Low bandwidth consistently takes 10s, high bandwidth 3 or 4s. The migration is repeated 3 times (defined as the last parameter in the calls to `repeatedlyMigrateWithBandwidthLimitation` here https://github.com/kubevirt/kubevirt/blob/main/tests/migration_test.go#L432-L433) in the test and the values are aggregated here https://github.com/kubevirt/kubevirt/blob/main/tests/migration_test.go#L421

> `*2.5` sounds good. I tried to use a value which did not fail after exhaustive runs on my machine (afaik I was far away from the case we see in CI) but still clearly indicated that a bandwidth limitation happened. A factor of `2.5` sounds good too from a validation perspective.

Cool thx, already pushed changes to update the factor

Reply:
Looks like in the failure cases we had 3 low bandwidth executions with 10s each (=30) and, for the high bandwidth, two executions of 3s and one of 4s ((3 + 3 + 4) * 3 = 30). If we decrease the factor to 2.5 we could afford 2 high bandwidth executions of 4s ((3 + 4 + 4) * 2.5 = 27.5) but not 3 ((4 + 4 + 4) * 2.5 = 30). Maybe we should decrease the factor a bit more?

Reply:
:+1: Let's take 2.

Reply:
ok done